scripts/minikube.sh

```diff
@@ -55,7 +55,17 @@ function install_minikube() {
         read -ra mk_version <<<"$(${minikube} version)"
         version=${mk_version[2]}
         echo "minikube already installed with ${version}"
-        return
+        mku_version=$(minikube update-check 2> /dev/null || ret=${?})
+        latest_version=$(echo ${mku_version} | cut -d' ' -f4)
+        if [[ ${ret} -ne 0 ]] || [[ -z ${latest_version} ]]; then
```

Comment:
Place variables in `"` quotes please. I guess `shellcheck` will complain about that too.

Reply:
sure will take care of this.